src/test/java/seedu/duke/commands/task/TaskListCommandTest.java

```diff
@@ -0,0 +1,35 @@
+package seedu.duke.commands.task;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.project.Project;
+import seedu.duke.project.ProjectList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class TaskListCommandTest {
+
+    @Test
+    void executeCommand() {
+        //Setup test project with existing tasks
+        ProjectList projects = new ProjectList();
+        projects.createProject("iPhone App");
+        projects.createProject("Android App");
+        int testIndex = 1;
+        Project iphone = projects.getProjectList().get(testIndex);
+        iphone.createTask("network features");
+        iphone.createTask("keyboard features");
+
+        //Expected Output
+        String expectedOutput = "";
+        expectedOutput += "List of Tasks:";
+        expectedOutput += "\n     " + "1" + "." + "network features";
+        expectedOutput += "\n     " + "2" + "." + "keyboard features";
```

Comment:
Would combining these statements into one single command be neater. Use line wrapping for the different lines of output.

Reply:
Yup thanks Sam